### ISSUE TITLE
Razor menu not rendering

### ISSUE THREAD

**Issue by DNNMonster** [nvisionative/nvQuickTheme#256]
## Describe the bug
Razor menu throws error on fresh DNN 9.9.1 install

## To Reproduce
1. Create a new DNN installation 
2. Follow the repo installation and build steps
3. View any page that is using the cloned and built NVQuickTheme
3. Review the Admin Logs

## Expected behavior
A fresh DNN installation and cloned NVQuickTheme repo should render a functional menu.  

## Errors
The issue results in logged errors and no visible menu

There error reads: 
\inetpub\wwwroot\storeo\Portals\_default\Skins\test\menus\razor\RazorMenu.cshtml(12): error CS0012: The type 'System.Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.

## Additional context
I am adding 2 entries to system.web\compilation\assemblies to enable the RazorMenu.cshtml to work.  If you add the first then you will get an additional error related to the netstandard assembly.  Things work fine once both are in the web config.

    <add assembly="System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
    <add assembly="netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />

I see some config mods happening in the manifest but I'm not clear if it would be best to enhance those to address these dependencies.

**Comment by david-poindexter** [comment]
@DNNMonster thanks for reporting this.  We will try this out and see if we can replicate the reported behavior.

**Comment by david-poindexter** [comment]
@DNNMonster I just tested and it is working fine for me.  Perhaps we can help narrow down, find, and resolve the issue for ya. 
 Here are a few questions:

 1. Did you modify anything or just build the cloned repo?  
 2. Did you build and just apply the theme to a page or the site?
 3. Did you try to package and install it on top of itself?
 
Let us know if there are any details to your steps that you find may be helpful for us to help you.  Thanks.

**Comment by DNNMonster** [comment]
1. I just cloned the repo.  Did the yarn magic. gulp build.
2. I am pretty sure I applied it to the site and also to pages.
3. I did not install the module on top of itself.

I've got it functioning so perhaps it was environmental or some other operator error.  For the dev machine I'm working on the menu fails if either of those entries are not in the web config.

If you're not reproducing it then let's close it up and it'll be here for reference if another person should have similar issues.
